1_Mechanics/02_GMGuide.md

```diff
@@ -706,7 +706,7 @@ If you want to make your own custom creature, you can use the sample template pr
 
 - **HP:** X
```

Comment:
Is this md format how you would want the beastiary to render? I split up attack mechanics into separate sections that would need to be recombined when rendering as markdown.

Reply:
Updated doc to be less detailed pending additional scoping of Powers List and other .md functions of the Bestiary.